examples/catalog-module/index.js

```diff
@@ -1,13 +1,20 @@
-import {render} from '../../src/index';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Catalog} from '../../src/index';
 
-render({
-  title: 'My Components',
-  pages: [
-    {
-      path: '/',
-      title: 'Foo',
-      imports: {Foo: require('./components/Foo/Foo')},
-      component: require('./components/Foo/Foo.docs.md')
-    }
-  ]
-}, document.getElementById('app'));
+ReactDOM.render(
+  <Catalog
+    configuration={{
```

Comment:
My first thought was that if we want to take the Catalog-as-component approach, it probably should take props individually and not all in one `configuration` prop, i.e. `<Catalog title='…' pages={[…]} … />`. But then we lose the property that the configuration is so nicely nestable (`{pages: [{pages: [{}]}]}`), although we're already cheating a little bit there as there are additional props on the top level that pages further down can't have (e.g. `basePath`).

The obvious next step would be:

```
<Catalog title='My Components'>
  <Page path='/' title='Foo' imports={{Foo: …}} component={…} />
  <Page title='Bar'>
      <Page path='/bla' title='Bla' imports={{Bla: …}} component={…} />
       <Page path='/blu' title='Blu' imports={{Foo: …}} component={…} />
  </Page>
</Catalog
```

Which brings us into React Router land and we know that it has both up- and downsides. So I'm currently undecided here …